Source/WebCore/dom/EventListenerMap.h

```diff
@@ -74,6 +74,16 @@ class EventListenerMap {
             callback(entry.first, entry.second.size());
     }
 
+    template <typename CallbackType>
+    bool containsMatchingEventListener(CallbackType callback)
```

Comment:
Would be nice to mark this function as `const`.

Reply:
Indeed this can be const.